src/test/java/com/google/cloud/tools/managedcloudsdk/ChildProgressListenerTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class ChildProgressListenerTest {
+
+  @Mock ProgressListener mockParent;
+
+  @Before
+  public void initMocks() {
```

Comment:
You prefer this over annotating the class with `@RunWith(MockitoJUnitRunner.class)`?

Reply:
Yeah, I don't know why though. I must have run into something weird in the past?